src/hid/hid-device.cpp

```diff
@@ -384,5 +430,12 @@ namespace librealsense
 
             return ep;
         }
+
+        void rs_hid_device::set_gyro_scale_factor(double scale_factor) 
+        {
+            _gyro_scale_factor = scale_factor;
+            if( scale_factor == 10000.0 )
+                _realsense_hid_report_actual_size = 38;
```

Comment:
Can't we use some `sizeof` here?
instead of hard coded value

Reply:
No because sizeof will show the size of the struct which will always be 38 bit